p2p/node.go

```diff
@@ -319,13 +319,6 @@ func (n *Node) ID() peer.ID {
 // Start causes the Node to continuously send messages to and receive messages
 // from its peers. It blocks until an error is encountered or `Stop` is called.
 func (n *Node) Start() error {
-	// Note: dht.Bootstrap has a somewhat confusing name. It doesn't automatically
```

Comment:
Calling `Bootstrap` is no longer necessary in the latest version of `go-libp2p-kad-dht`. See https://github.com/libp2p/go-libp2p-kad-dht/releases/tag/v0.3.0.